src/screens/qr/views/QRCodeScanner.tsx

```diff
@@ -45,7 +44,7 @@ export const QRCodeScanner = () => {
     >
       <View style={styles.layerTop} />
       <Box style={styles.back} paddingHorizontal="m">
-        <ToolbarWhite isWhite={true} />
+        <ToolbarWhite isWhite />
```

Comment:
If we're not going to use `BaseQRCodeScreen` on the only screen that needs the white text toolbar, let's remove the `isWhite` prop from `BaseQRCodeScreen` and hard code it to use black text on the toolbar.